### PR TITLE
Add New contributor summit notes

### PR DESCRIPTION
**Pull request by jgsqware**
I took some notes during the New contributor summit notes.

Here they are

### CODE REVIEW

File: contributors/new-contributor-playground/new-contributor-nodes.md [hunk 1]
```diff
@@ -0,0 +1,346 @@
...
+### Kubernetes docs
+
+- Websites URL
+- Github Repository
+- k8s slack: #sig-docs
+
+### Working with docs
+
+Docs use `k8s-ci-robot`. Approval process is the same as for any k8s repo
+In docs, `master` branch is the current version of the docs. So always branch from `master`. It's continous deployment
```
Comment (guineveresaenger) [inline contributors/new-contributor-playground/new-contributor-nodes.md]: "continuous"

File: contributors/new-contributor-playground/new-contributor-nodes.md [hunk 2]
```diff
@@ -0,0 +1,346 @@
...
+    - sig-release
+    - sig-testing: In charge of all the test for Kubernetes
+5. Docs
+    - sig-docs: for documentation and website
+
+## Working groups and Subproject
+
+From working group to subproject. 
+
+for stuff really specific. Eg. `Helm` Belong to *SIG-apps* but dedicated subporject
```
Comment (guineveresaenger) [inline contributors/new-contributor-playground/new-contributor-nodes.md]: "subproject"

**Comment by guineveresaenger** [comment]
These are wonderful notes; thank you!

File: contributors/new-contributor-playground/new-contributor-nodes.md [hunk 3]
```diff
@@ -0,0 +1,346 @@
+# New kubernetes contributor workshop
```
Comment (smourapina) [inline contributors/new-contributor-playground/new-contributor-nodes.md]: I would change this to: 
```
# Kubernetes New Contributor Workshop - KubeCon EU 2018
```

File: contributors/new-contributor-playground/new-contributor-nodes.md [hunk 4]
```diff
@@ -0,0 +1,346 @@
...
+
+Joining in the beginning was onboarding on a yacht
+Now is more onboarding a BIG --Paquebot
+
+Will be a Hard schedule, and let's hope we can achieve everything
+Sig-contributor-experience -> from Non-member contributores to Owner
```
Comment (smourapina) [inline contributors/new-contributor-playground/new-contributor-nodes.md]: Typo: *contributors

File: contributors/new-contributor-playground/new-contributor-nodes.md [hunk 5]
```diff
@@ -0,0 +1,346 @@
...
+- SIG-\* (*depends on the area to contribute on*): **Code** contribution
+
+**=> Find your first topics**: bug, feature, learning, community development and documentation
+
+Table exercise: Introduce yourself and give a tip on where you want to contribute in Kubernetes
+
+
+## Communication in the community
+
+Kubernetes community is like a Capybara: community members are really cool with everyone and they are from a lot of diffrenet horizons.
```
Comment (smourapina) [inline contributors/new-contributor-playground/new-contributor-nodes.md]: Typo: *different

File: contributors/new-contributor-playground/new-contributor-nodes.md (hunk 2 above)
Comment (smourapina) [reply contributors/new-contributor-playground/new-contributor-nodes.md]: I would change this text to:
```
For specific: tools (ex. Helm), goals (ex. Resource Management) or areas (ex. Machine Learning).
```

Comment (smourapina) [reply contributors/new-contributor-playground/new-contributor-nodes.md]: I would also add that groups change around more frequently than SIGs, and some might be temporary.

File: contributors/new-contributor-playground/new-contributor-nodes.md [hunk 6]
```diff
@@ -0,0 +1,346 @@
...
+
+Move faster than SIG 
+
+- wg-app-def
+- wg-apply
+- wg-cloud-provider
+- wg-cluster-api
+- wg-container-identity
+- ...
+
```
Comment (smourapina) [inline contributors/new-contributor-playground/new-contributor-nodes.md]: Here you could add:
```
### Picking the right SIG:
1. Figure out which area you would like to contribute to
2. Find out which SIG / WG / subproject covers that (tip: ask on #sig-contribex Slack channel)
3. Join that SIG / WG / subproject (you should also join the main SIG when joining a WG / subproject)
```

File: contributors/new-contributor-playground/new-contributor-nodes.md [hunk 7]
```diff
@@ -0,0 +1,346 @@
...
+## Tour des repositories
+
+Everything will be refactored (cleaning, move, merged,...)
+
+### Core repository
```
Comment (smourapina) [inline contributors/new-contributor-playground/new-contributor-nodes.md]: I would add a brief description: this is where the core code is.

File: contributors/new-contributor-playground/new-contributor-nodes.md [hunk 8]
```diff
@@ -0,0 +1,346 @@
...
+```
+- Bug or Feature
+
+- What happened
+
+- How to reproduce
+
+```
+
+### Issue for specification
```
Comment (smourapina) [inline contributors/new-contributor-playground/new-contributor-nodes.md]: I would change to 
```
### Issues as specification
```

File: contributors/new-contributor-playground/new-contributor-nodes.md (hunk 2 above)
Comment (jgsqware) [reply contributors/new-contributor-playground/new-contributor-nodes.md]: Thanks, updated

File: contributors/new-contributor-playground/new-contributor-nodes.md (hunk 1 above)
Comment (jgsqware) [reply contributors/new-contributor-playground/new-contributor-nodes.md]: Thanks, updated

File: contributors/new-contributor-playground/new-contributor-nodes.md (hunk 3 above)
Comment (jgsqware) [reply contributors/new-contributor-playground/new-contributor-nodes.md]: :+1:

**Comment by jgsqware** [comment]
@guineveresaenger I updated with your comment. thanks

**Comment by guineveresaenger** [comment]
/lgtm

Thanks @smourapina as well for detailed review! <3

**Pull request merged**